### ISSUE TITLE
2차 세미나와 snupc대회가 겹칩니다.

### ISSUE THREAD

**Issue by pasa3232** [wafflestudio/rookies#113]
9월 12일 토요일 2차 세미나 일정이 잡혀있는 시간에 snups동아리에서 진행하는 snupc대회가 오후 1시부터 5시까지 진행됩니다.
snupc대회로 인한 세미나 불참이 적절한 불참 사유가 될 수 있을지 궁금합니다!

**Comment by Hank-Choi** [comment]
네 snupc 참가로인해 세미나를 빠지는 분들은 저 또는 각 세미나장에게 개인톡을 보내주시면 반영하고 적절한 대체 방법을 생각해보겠습니다~ 👍
